src/utils/fields.js

```diff
@@ -80,9 +79,15 @@ const FOLDER_FIELDS_TO_FETCH = [
     FIELD_AUTHENTICATED_DOWNLOAD_URL,
     FIELD_IS_DOWNLOAD_AVAILABLE,
     FIELD_REPRESENTATIONS,
+    FIELD_URL,
+];
+
+// Fields needed for Content Explorer folder requests
+const CONTENT_EXPLORER_FOLDER_FIELDS_TO_FETCH = [
```

Comment:
Should this constant live in a file within the `src/elements/content-explorer` directory?

Reply:
Yes! Just moved the constant into the `src/elements/content-explorer` directory.